tests/pytest_tests/system_tests/test_notebooks/test_notebooks.py

```diff
@@ -259,21 +260,34 @@ def test_code_saving(notebook):
         assert "WANDB_NOTEBOOK_NAME should be a path" in nb.all_output_text()
 
 
-@pytest.mark.skip_wandb_core(feature="launch")
 def test_notebook_creates_artifact_job(notebook):
     with notebook("one_cell_disable_git.ipynb") as nb:
         nb.execute_all()
         output = nb.cell_output_html(2)
-        # 3 artifact files if code, 5 if also job
-        # TODO: test contents of artifact, requires relay server context for artifacts
-        assert "5 artifact file(s)" in output
+    regex_string = r'http:\/\/localhost:8080\/[^\/]+\/[^\/]+\/runs\/([^\'"]+)'
+    run_id = re.search(regex_string, str(output)).group(1)
+
+    api = wandb.Api()
+    user = os.environ["WANDB_USERNAME"]
+    run = api.run(f"{user}/uncategorized/{run_id}")
+    used_artifacts = run.used_artifacts()
+    assert len(used_artifacts) == 1
+    assert (
+        used_artifacts[0].name
+        == "job-source-uncategorized-one_cell_disable_git.ipynb:v0"
+    )
```

Comment:
do you think you can achieve something similar with the relay server? just wanted to check

Reply:
I tried the relay server and afaict executing subprocesses within the context of the relay server doesn't work

Reply:
oh, weird can you open an issue for us? i know @dmitryduev verified that pyrun works surprised about the subprocess :/

Reply:
My mistake, *notebooks* dont seem to hit the relay server, not sure about subprocesses.

Reply:
super weird